docs/ops/arithmetic/Round_5.md

```diff
@@ -0,0 +1,57 @@
+## Round <a name="Round"></a> {#openvino_docs_ops_arithmetic_Round_5}
+
+**Versioned name**: *Round-5*
+
+**Category**: Arithmetic unary operation 
+
+**Short description**: *Round* performs element-wise round operation with given tensor.
+
+**Detailed description**: Operation takes one input tensor and rounds the values, element-wise, meaning it finds the nearest integer for each value. In case of halfs, the rule is to round them to the nearest even integer if `mode` attribute is `half_to_even` or rounding in such a way that the result heads away from zero if `mode` attribute is `half_away_from_zero`.
+    
+    Input = [-4.5, -1.9, -1.5, 0.9, 1.5, 2.3, 2.5]
+    
+    round(Input, mode = `half_to_even`) = [-4.0, -2.0, -2.0, 1.0, 2.0, 2.0, 2.0]
```

Comment:
Please, add a case when the input value is 0.5 to better see the difference between modes.